### ISSUE TITLE
CTRL + left/right does not work in text fields/textareas

### ISSUE THREAD

**Issue by tmysik** [OtterBrowser/otter-browser#843]
IMHO this is a **regression** in Otter 0.9.06 beta 6. Steps:

- have a page with a textarea
- type some words in it
- keep cursor (focus) in this textarea
- try to navigate among the words using CTRL + <arrow left> and CTRL + <arrow right>

-> expected: cursor "jumps" over the words
-> actual: browser navigates back and forward in its history (if it is a new tab, nothing happens)

Thanks.

**Comment by beastie1** [comment]
This can probably be considered as a duplicate of #826.

A quick temporary fix is to remove the two keyboard shortcuts from **Tools > Preferences > Advanced > Keyboard > Edit "Default" > Back/Forward**

**Comment by paradogz** [comment]
Still an issue. I guess it is now blocked by #926.

**Comment by Emdek** [comment]
@paradogz, which backend you are using? It was fixed long time ago in QtWebKit while QtWebEngine is not fixable at this time (no sane ability to detect if user is editing something).
